nuxt-frontend/components/ios-app-development/CaseStudySection.vue

```diff
@@ -78,35 +72,29 @@
           :mousewheel="{
             enabled: false,
             releaseOnEdges: true,
-            sensitivity: 1,
-            thresholdDelta: 1,
           }"
-          :touchReleaseOnEdges="true"
           :modules="modules"
           :allowTouchMove="false"
           class="swiper-container h-screen"
           @swiper="setSwiperRef"
-          @slideChange="onSlideChange"
         >
           <swiper-slide v-for="(portfolio, index) in portfolios" :key="index">
             <div class="flex flex-row justify-center h-screen">
               <div
                 :class="portfolio.row1Background"
                 class="flex flex-col justify-center !basis-[40%] justify-around px-8"
               >
-                <div class="flex justify-end relative">
+                <div class="flex justify-end relative text-[#FFFFFF]">
                   <div
                     :class="portfolio.row1Background"
-                    class="absolute top-4 xl:top-8 right-[-25%] xl:right-[-17%] 2xll:right-[-18%] xll:right-[-12%] 3xl:right-[-13%] text-[#FFFFFF] w-[250px] h-[250px] p-8 xl:w-[300px] xl:h-[300px] rounded-full flex justify-center items-center"
-                  >
-                    <h2 class="header-1 title">
-                      {{ portfolio.title }}
-                    </h2>
-                  </div>
+                    class="absolute top-4 xl:top-8 right-[-25%] xl:right-[-17%] 2xll:right-[-18%] xll:right-[-12%] 3xl:right-[-13%] w-[250px] h-[250px] p-8 xl:w-[300px] xl:h-[300px] rounded-full flex items-center"
+                  ></div>
+                  <h2 class="absolute top-32 header-1 title right-[-30px]">
+                    {{ portfolio.title }}
+                  </h2>
                 </div>
                 <div
                   class="w-[19rem] xl:w-[33rem] mx-auto mt-44 xl:mt-72 description"
-                  :class="{ '!mt-12 xl:!mt-72': isScrollingUp }"
                 >
                   <h2 class="header-2 text-[#FFFFFF]">
```

Comment:
- i think it's white color so you can directly use tailwind class
- also, `span` has same color with opacity so you can refactor it like give white color to parent and set opacity for child